src/trans-word-level/counterexample_word_level.cpp

```diff
@@ -37,7 +39,9 @@ void show_state(
 
   const symbol_tablet &symbol_table=ns.get_symbol_table();
   
-  forall_symbol_module_map(it, symbol_table.symbol_module_map, module)
+  for(auto it=symbol_table.symbol_module_map.lower_bound(module);
```

Comment:
Since we only want to visit the symbols with `module` we propably have to get the range using `upper` and `lower_bound`. And these return iterators without any simple way of getting the `begin` `end` functionality needed for ranged for loop.